### ISSUE TITLE
maven-repair:npefix crashes with dirty exception

### ISSUE THREAD

**Issue by monperrus** [eclipse/repairnator#1159]
See https://github.com/eclipse/repairnator/pull/1153

**Comment by andre15silva** [comment]
For some reason, `Lapse`s don't have any `Decision`, resulting in `executions` not being populated with anything.

I haven't yet been able to understand why this happens.

**Comment by monperrus** [comment]
An easy workaround would be to have an empty executions array by default.

**Comment by monperrus** [comment]
Is that fixed now that #1137 is merged?

**Comment by andre15silva** [comment]
No, the issue comes from `npefix` and hasn't been fixed yet. See https://github.com/SpoonLabs/npefix/issues/24
